charts/kube-prometheus-stack/UPGRADE.md

```diff
@@ -1,5 +1,22 @@
 # Upgrade
 
+## From 70.x to 71.x
+
+This version upgrades Prometheus-Operator to v0.82.0.
```

Comment:
```suggestion
This version upgrades Prometheus-Operator to v0.82.0
Since [68.4.0](https://github.com/prometheus-community/helm-charts/pull/5175) it is also possible to use `crds.upgradeJob.enabled` for upgrading the CRDs.
For traditional upgrades, please run these commands to update the CRDs before applying the upgrade.
```